actions/chains/bgp_prefix_exceeded_chain.yaml

```diff
@@ -0,0 +1,75 @@
+---
+  chain:
+      -
+        name: "show_bgp_neighbours"
```

Comment:
So I can see we're going to get into an argument here with our American English-speaking cousins. How ready are you to defend neighbour vs neighbor? :trollface: 

(For reference: I grew up speaking New Zealand English, which is much closer to the Queen's English than this derivative I now battle on a daily basis)